test/e2e/admin_portal.go

```diff
@@ -244,4 +244,36 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 			Expect(link.GetAttribute("href")).To(MatchRegexp(`https://([a-z]|[0-9])+\.admin\.aro\.azure\.com`))
 		}
 	})
+
+	It("Should open an error modal for an invalid resource ID parameter in the URL", func() {
+		wd.Get(host + "/v2" + "?resourceid=" + "invalidResourceId")
+		time.Sleep(time.Second * 2) // this is done to give enough time to fetch the data
+
+		wd.Wait(conditions.ElementIsLocated(selenium.ByCSSSelector, "div[role='document']"))
```

Comment:
We shouldn't have both time.Sleep and wd.Wait. We should be waiting with specific enough selectors that it executes the test when the dom is ready with the data fetched.